websites/D/Discord Resources/dist/metadata.json

```diff
@@ -9,10 +9,10 @@
 		"en": "A list of tons of useful Discord resources and utilities for all types of users, from beginners to power users."
 	},
 	"url": "discordresources.com",
-	"version": "1.0.0",
+	"version": "1.0.1",
 	"logo": "https://i.imgur.com/LzdNPBJ.png",
 	"thumbnail": "https://i.imgur.com/LCBideV.png",
-	"color": "#1ed760",
+	"color": "#16c2ab",
```

Comment:
this is far from purple

Reply:
Changed it to the suggested color

Reply:
There’s a button to commit the suggestion yet you managed to do that wrong 🤔